corehq/apps/app_manager/helpers/validators.py

```diff
@@ -576,6 +578,18 @@ def _validate_fields_with_format(
                             .format(format_display, fields_with_address_format_str))
             })
 
+    def _validate_clickable_icons(
+        self,
+        columns: list,
+        errors: list
+    ):
+        for field in [c.field for c in columns if c.format == 'clickable-icon' and c.endpoint_action_id == '']:
```

Comment:
Does the check for an empty string cover all cases? Is `null` not a possible value? Just checking for a empty string may be correct, but I couldn't say for sure.

Reply:
If you don't select a form on the config page with will set it to ""